sysdiglabs/grok_exporter
# Fixes for security

[The latest image has a few security issues in it](https://quay.io/repository/sysdig/grok-exporter?tab=tags), which this PR tries to address

* Switch to latest golang and use the builder image
* Use distroless image for running in. Note that this has to match the builder to get the right C libs.
* Upgrade onig-6.9.5-rev1 -> onig-6.9.9
* Update all packages with go get -u
  * This means renaming github.com/Shopify/sarama -> ithub.com/IBM/sarama
  * Also found an erroneous usage of promehteus logging which I switched to logrus as it seemed the original intention
* Removed vendoring

These changes are [published to docker hub, for which scout currently reports no vulnerabilities](https://hub.docker.com/layers/vsliouniaev/grok-exporter/v1.0.7/images/sha256-63afc6b9f33f0b8bfee798eb2a8f0f76a498074606ea1983df9ae48b59008e31?context=repo)
